admin/components/Home/GettingStarted.js

```diff
@@ -24,7 +25,7 @@ const GettingStarted = (props, { t }) => {
                     <span>
                         <i className="fa fa-eye" />
                     </span>
-                    <Link to="/admin/post-new">{t("home.view")}</Link>
+                    <Link to={config.baseName + "/"}>{t("home.view")}</Link>
```

Comment:
We will not need `config.baseName`.  If you look at https://github.com/letterpad/letterpad/blob/520651758bc362e03a4bc7f10f2a7ee309850743/client/app.js#L10 we are adding the basename on the router. So Link will automatically handle the basename for us. Just make it `"/"` and that should work.

Reply:
@ajaxtown missed this. Thanks. Fixed.